modular_skyrat/modules/specialist_armor/code/cargo_packs.dm

```diff
@@ -0,0 +1,32 @@
+/datum/supply_pack/security/armory/soft_armor
+	name = "Soft Armor Kit Crate"
+	crate_name = "soft armor kit crate"
+	desc = "Contains three sets of SolFed-made soft body armor and matching helmets."
+	access = ACCESS_MEDICAL
+	cost = CARGO_CRATE_VALUE * 5
+	contains = list(
+		/obj/item/clothing/head/helmet/sf_peacekeeper/debranded = 3,
+		/obj/item/clothing/suit/armor/sf_peacekeeper/debranded = 3,
+	)
+
+/datum/supply_pack/security/armory/hardened_armor
+	name = "Hardened Armor Kit Crate"
+	crate_name = "hardened armor kit crate"
+	desc = "Contains three sets of SolFed-made hardened body armor and matching helmets."
+	access = ACCESS_MEDICAL
+	cost = CARGO_CRATE_VALUE * 5
```

Comment:
This is the same exact cost as the soft armor vests, why would anyone go for a soft armor vest over a hardened one?

Reply:
They have significantly higher wound protection than the hardened vests do, twice as much as mentioned earlier.